### ISSUE TITLE
Cannot get the *nix vagrant builds to work

### ISSUE THREAD

**Issue by ibell** [wxWidgets/Phoenix#1023]
I think the instructions for the vagrant builds are no longer up to date.  I tried to follow the instructions verbatim, and got as far as https://github.com/wxWidgets/Phoenix/tree/master/vagrant#setup without problems.

As soon as I hit the build steps, all hell broke loose.  I was unable to get the command ``build.py dox etg sip sdist`` to work (probably it should actually be ``python build.py dox etg sip sdist`` ?).  And it was all downhill from there.  Since i couldn't generate the dist, I couldn't get the vagrant builds to run.  Now that I think about it, perhaps the approach would be to do a ``pip download wxPython`` in the ``{PhoenixRoot}/dist`` folder?

**Comment by RobinD42** [comment]
Yes that should work as well. As the readme says you can use a source tarball from a release or a prerelease snapshot. How you get or generate the source depends on your needs and the reason you are doing the build.

What errors/problems did you have with running `build.py` to make the source? Did you run it from the PhoenixRoot folder? Are you using the master branch or the wxPy-4.0.x branch?

**Comment by ibell** [comment]
Hmm, yeah, sorry about the non-descriptive bug report.  I've gotten the
builds going now with a "pip download wxPython" in the {PhoenixRoot}/dist
folder, but man, are they slow.  Is there any hope of getting a slightly
more speedy update schedule for the wheels at
https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-18.04/?
It's been three months since the last build (or so).

Once these builds complete I'll try to run the build.py script again and
give you the error log.

**Comment by ibell** [comment]
Indeed, that all works fine now with the vagrant builds, but I still get this error:
```
(py27) ian@ian-ThinkPad-X220:~/Code/Phoenix$ python build.py sdist
Will build using: "/home/ian/miniconda3/envs/py27/bin/python"
2.7.15 |Anaconda, Inc.| (default, May  1 2018, 23:32:55) 
[GCC 7.2.0]
Python's architecture is 64bit
cfg.VERSION: 4.1.0a1

Running command: sdist
Exporting Phoenix...
Exporting wxWidgets...
fatal: current working directory is untracked
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors
Command '(cd ext/wxWidgets; git archive HEAD) | tar -x -C build/sdist/ext/wxWidgets' failed with exit code 2.
Finished command: sdist (0.415s)
```
